File: CookesMafia/CookesMafia/Models/Person.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace CookesMafia.Models
{
    public class Person
    {
        public int Id { get; set; }
        public string Name { get; set; } 
        public double Money { get; set; }
        public double Alldebit { get; set; }
        [InverseProperty("Person")]
        public  List<Debit> Debites { get; set; }
        [InverseProperty("Borrower")]
        public  List<Debit> Borrowes { get; set; }

        public Person()
        {
            Debites = new List<Debit>();
            Borrowes = new List<Debit>();
        }
    }
}

File: ReactClient/cookie-client/src/Models/Person.ts
import Debit from "./Debit";

export default class Person{
    public id:number = 0;
    public name:string = ""; 
    public money:number = 0; 
    public debites:Debit[] =[];
    public borrowes:Debit[] =[];
    public alldebit:number=0;
    
}